crates/sui-core/src/checkpoints/checkpoint_executor/mod.rs

```diff
@@ -436,7 +436,7 @@ impl CheckpointExecutor {
 
     /// Post processing and plumbing after we executed a checkpoint. This function is guaranteed
     /// to be called in the order of checkpoint sequence number.
-    #[instrument(level = "debug", skip_all)]
+    #[instrument(level = "info", skip_all, fields(seq = ?checkpoint.sequence_number()))]
```

Comment:
did you mean to merge this "info" level bump?

Reply:
yes, once-per-checkpoint spans are good candidates for info level imo